src/com/rcv/Tabulator.java

```diff
@@ -55,6 +58,7 @@
     this.castVoteRecords = castVoteRecords;
     this.candidateIDs = config.getCandidateCodeList();
     this.config = config;
+    initPrecinctRoundTallies();
```

Comment:
Might be more optimized if we check to see in the first place if the config specifies that we should group by precinct before initializing this.

Reply:
There's no specific setting for this in the config; there's just an optional field to indicate which column has the precinct in the CVR file. But if no CVRs have precincts, the precinct tallying logic will basically be a no-op... so I think it's fine?

Reply:
Hmm, I might be wrong but looks like it's still O(n) where n is size of castVoteRecords. Maybe a good idea to create an issue to add this option as something we can specify in the config file in the future? I'll leave it up to you.

Reply:
I'll ask if we need a global setting for tabulate by precincts (my guess is yes).  Performance isn't an issue (I doubt it will ever be) but more for clarity in the code.

Reply:
OK, I opened an issue for this. Seems reasonable to have a setting for it.